3 Domain layer/CandidatesEvaluator.Contract/Queries/QueryBase.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CandidateEvaluator.Contract.Queries
+{
+    public class QueryBase
```

Comment:
This one class is redundant. OwnerId prop could be placed in all queries. There is no gain to extract it as a base class.